src/scripts/StoreTalismanApi.js

```diff
@@ -0,0 +1,77 @@
+import axios from 'axios';
+import StoreRestApi from './StoreRestApi';
+import urlJoin from 'url-join';
+
+export default class StoreTalismanApi extends StoreRestApi {
+	constructor(config, baseUrl) {
+		const sitemapsPath = urlJoin('api', 'sitemaps/');
+		super(config, baseUrl, sitemapsPath);
+	}
+
+	async initTalismanLogin(credentials) {
+		let bodyForm = new FormData();
+		let tLogin = credentials.username;
+		let tPassword = credentials.password;
+		bodyForm.append('username', tLogin);
+		bodyForm.append('password', tPassword);
+		let loginStatus = await this.axiosInstance.post(
```

Comment:
Throws error on invalid credentials which replaces response in background.js and is not handled correctly.